cabby/data/wikidata/query.py

```diff
@@ -0,0 +1,197 @@
+# coding=utf-8
+# Copyright 2020 Google LLC
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+'''Library to support Wikidata areal queries.'''
+
+import sys
+from SPARQLWrapper import SPARQLWrapper, JSON
+from typing import Dict, Tuple, Sequence, Text
+
+_MANHATTAN_QUERY = """SELECT ?place ?placeLabel (GROUP_CONCAT(DISTINCT
```

Comment:
could you add a reference to the Wiki API in the file comment for reference?
It is https://www.mediawiki.org/wiki/API:Main_page I guess? Or may be there is a more specific page that is relevant to us.

Reply:
I query over - https://query.wikidata.org/sparql added to the comments